1주차/14659/14659_python_후디.py

```diff
@@ -0,0 +1,28 @@
+n = int(input())
+attackers = list(map(int, input().split(" ")))
+
```

Comment:
TIP: 저의 경우 저 부분은 생략해요. 디폴트가 `" "`라서요

```suggestion
attackers = list(map(int, input().split()))
```

Reply:
아! 오랜만에 파이썬을 잡으니 디폴트 값을 잊었네요 ㅎㅎ! 감사합니다 👍